.golangci.yaml

```diff
@@ -16,6 +15,7 @@ linters:
     - goimports
   disable:
     - errcheck
+    - prealloc
```

Comment:
Because forcing allocating memory for a slice is just wrong. There are cases where this is desirable and cases where it does not apply. Not allocating results in variable set to `nil` which is a perfectly valid value to distinguish it from an empty slice.

Reply:
I don't think this should have being changed in this PR, there was enough other things going on. I see your agreement but I don't agree with it. I think it was a language mistake that a slice could be a nil pointer after creation.

Reply:
If this doesn't change in this PR, then code style check fails at this line: https://github.com/Kuadrant/kuadrant-operator/blob/479d49bcc96a8b3889a8900e08337c35744e9498/controllers/authconfigs_reconciler.go#L68.